internal/cmd/perfcomp/main.go

```diff
@@ -305,27 +318,31 @@ func getEnergyStatsForAllBenchMarks(ctx context.Context, patchRawData []RawData,
 
 func generatePRComment(energyStats []*EnergyStats, version string) string {
 	var comment strings.Builder
-	comment.WriteString("# 👋GoDriver Performance\n")
-	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n", version)
+	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n\n", version)
 
 	w := tabwriter.NewWriter(&comment, 0, 0, 1, ' ', 0)
 	fmt.Fprintln(w, "| Benchmark\t| Measurement\t| H-Score\t| Z-Score\t| % Change\t| Stable Reg\t| Patch Value\t|")
 	fmt.Fprintln(w, "| ---------\t| -----------\t| -------\t| -------\t| --------\t| ----------\t| -----------\t|")
 
-	var testCount int64
+	var significantEnergyStats []EnergyStats
 	for _, es := range energyStats {
-		if math.Abs(es.ZScore) > 1.96 {
-			testCount += 1
-			fmt.Fprintf(w, "| %s\t| %s\t| %.4f\t| %.4f\t| %.4f\t| Avg: %.4f, Med: %.4f, Stdev: %.4f\t| %.4f\t|\n", es.Benchmark, es.Measurement, es.HScore, es.ZScore, es.PercentChange, es.StableRegion.Mean, es.StableRegion.Median, es.StableRegion.Std, es.MeasurementVal)
+		if es.Measurement != "iterations" && math.Abs(es.ZScore) > 1.96 {
```

Comment:
Optional: Add a comment that describes why we're skipping the "iterations" measurements.

E.g.
```go
// The "iterations" measurement is the number of iterations that the Go
// benchmark suite had to run to converge on a benchmark measurement. It
// is not comparable between benchmark runs, so is not a useful
// measurement to print here. Omit it.
```